src/Adapter/Driver/Oci8/Result.php

```diff
@@ -6,73 +6,66 @@
  * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
  * @license   http://framework.zend.com/license/new-bsd New BSD License
  */
-
 namespace Zend\Db\Adapter\Driver\Oci8;
 
 use Iterator;
 use Zend\Db\Adapter\Driver\ResultInterface;
 use Zend\Db\Adapter\Exception;
-
 class Result implements Iterator, ResultInterface
 {
     /**
      * @var resource
      */
     protected $resource = null;
-
     /**
-     * @var bool
+     * @var null
      */
-    protected $isBuffered = null;
-
+    protected $rowCount = null;
     /**
      * Cursor position
      * @var int
      */
     protected $position = 0;
-
     /**
      * Number of known rows
      * @var int
      */
     protected $numberOfRows = -1;
-
     /**
      * Is the current() operation already complete for this pointer position?
      * @var bool
      */
     protected $currentComplete = false;
-
     /**
      * @var bool
      */
     protected $currentData = false;
-
     /**
      *
      * @var array
      */
     protected $statementBindValues = ['keys' => null, 'values' => []];
-
     /**
      * @var mixed
      */
     protected $generatedValue = null;
-
     /**
      * Initialize
      * @param resource $resource
+     * @param null|int $generatedValue
+     * @param null|int $rowCount
      * @return Result
      */
-    public function initialize($resource /*, $generatedValue, $isBuffered = null*/)
+    public function initialize($resource, $generatedValue = null, $rowCount = null)
```

Comment:
See above. The body of the method was ignoring it already, and the argument is not defined in the governing interface (which is why it's an optional argument in the concrete definition).